source/src/main/groovy/com/kezong/fataar/VariantProcessor.groovy

```diff
@@ -320,7 +320,9 @@ class VariantProcessor {
         ManifestProcessorTask processManifestTask = mVersionAdapter.getProcessManifest()
 
         File manifestOutput
-        if (FatUtils.compareVersion(VersionAdapter.AGPVersion, "3.3.0") >= 0) {
+        if (FatUtils.compareVersion(VersionAdapter.AGPVersion, "4.2.0-alpha07") >= 0) {
```

Comment:
Are you checked it in 4.2.0-alpha01 ~ 4.2.0-alpha06 ?

Reply:
Yes, totally. Manually checked all of them.